src/components/Search/command.tsx

```diff
@@ -1,14 +1,23 @@
-import { useState } from 'react';
 import { Clipboard } from 'lucide-react';
 import { DiceCmdMeta } from '@/data/command';
 
-export default function CommandPage({ title, syntax, body, url }: DiceCmdMeta) {
```

Comment:
this seems unrelated to static site generation, and it breaks the behaviour of "Copied" showing up is that expected?